pulsar-common/src/main/java/org/apache/pulsar/common/util/keystoretls/KeyStoreSSLContext.java

```diff
@@ -38,6 +38,7 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.pulsar.common.util.SecurityUtility;
+import org.apache.pulsar.common.util.jetty.JettySslContextFactory;
```

Comment:
Drop references to the Jetty support since pulsar-common is also used on the client. no need to have a deprecation cycle since this is not a public API for Pulsar.

Reply:
Fixed.